index.Rmd

```diff
@@ -5,14 +5,14 @@ subtitle: An interactive tool for understanding the distribution of invasive ins
 output: html_document
 ---
 
-# Introduction
+# Introduction 
 Invasive species are organisms that cause ecological or economic harm in environments in which they are not native. As globalization has spread and the world has grown exponentially smaller, invasive species have become one of the greatest challenges facing environmental managers today. Coupled with constant influx of imported goods is the ever-looming spectre of climate change. Biodiversity loss caused by climate change may leave room in an ecosystem in which the invader has no natural predators, causing species hierarchies to shift, and the invasive species to outcompete native species already vulnerable due to climate induced stress. While there are numerous other factors involved in the spread of invasive species, this map seeks to ask the question: **Is the spread of invasive species correlated with an increase in mean temperature? Land usage or land cover?** This project aims to build an interactive tool that allows users to trace the spread of numerous invasive species in order to further understand the factors that allow invasive species to spread.  
 
-# Materials and methods
+# Materials and methods 
```

Comment:
Once you finalize them, fill this in with more details about your specific methods.